objects/vcs/github.rb

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2016-2021 Yegor Bugayenko
```

Comment:
@mbao01 this file is in this PR by mistake?

Reply:
@yegor256 No, it is not. It will be used in the next PR. Trying to avoid a large PR.

Reply:
@mbao01 we don't do this. we make sure that ALL changes required in a PR stay in the PR. and we don't put anything that doesn't belong to it.

Reply:
@yegor256 okay, I will close this one. I combined the changes here with the next PR.